pyang/statements.py

```diff
@@ -1785,6 +1785,22 @@ def add_tmp_children(node, tmp_children):
                 node.i_children.append(tmp)
                 tmp.parent = node
 
+    def keyword_is_expected(pos, parent, child):
```

Comment:
I'd prefer `is_keyword_expected`

Reply:
Sure. I will update it by your comments.